tutorials/variational-inference/01_variational-inference.jmd

```diff
@@ -351,7 +219,7 @@ p_μ_pdf = z -> exp(logpdf(p_μ, (z - μₙ) * exp(- 0.5 * log(βₙ) + 0.5 * lo
 p1 = plot();
 histogram!(samples[1, :], bins=100, normed=true, alpha=0.2, color = :blue, label = "")
 density!(samples[1, :], label = "s (ADVI)", color = :blue)
-density!(collect(skipmissing(samples_nuts[:s].value)), label = "s (NUTS)", color = :green)
+density!(vec(samples_nuts[:s].data), label = "s (NUTS)", color = :green)
```

Comment:
Same question here?

Reply:
You mean just do `vec(samples_nuts[:s])`? If so, yeah! I did this a bit quickly and just assumed `vec` didn't work, but it's an `AxisArray` now so ofc.

Reply:
Hmm is `samples_nuts` not a Chains object? I though either we should use something like `Array(samples_nuts[:s])` (or just `samples_nuts[:s]`) (maybe with `vec` if needed) or just `density!(samples_nuts, :s, ...)` (or something similar) since we define all these recipes in MCMCChains.

Reply:
> density!(samples_nuts, :s, ...)

Wait, WHAT?! How come I don't know about this?! That's awesome :+1: I'll use those then !

Reply:
Haha make sure to check if/how it works first and don't rely on my vague memories :smile:

Reply:
It worked:)